src/Controls/src/Core/Handlers/Shapes/Rectangle/RectangleHandler.Standard.cs

```diff
@@ -5,6 +5,7 @@ namespace Microsoft.Maui.Controls.Handlers
 {
 	public partial class RectangleHandler
 	{
+		internal static void MapBackground(IShapeViewHandler handler, Rectangle rectangle) { }
```

Comment:
In the case of the Rectangle, `BackgroundColor`, `Background` and `Fill` changes should perform the same task, and fill the background color.